content/docs/instrumenting/exporters.md

```diff
@@ -64,6 +64,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [NVIDIA GPU exporter](https://github.com/mindprince/nvidia_gpu_prometheus_exporter)
    * [ProSAFE exporter](https://github.com/dalance/prosafe_exporter)
    * [Ubiquiti UniFi exporter](https://github.com/mdlayher/unifi_exporter)
+   * [Netgear Cable Modem Exporter](https://github.com/ickymettle/netgear_cm_exporter)
```

Comment:
Please keep the list alphabetically sorted.

Reply:
Thanks @juliusv, fixed!